doc/AST_FORMAT.md

```diff
@@ -864,6 +864,19 @@ Format:
  ~~~ name
 ~~~
 
+### Required single block argument
```

Comment:
Just put the version requirement in the title, like `### Required single block argument (1.9)`. This is the convention used elsewhere.